package.json

```diff
@@ -450,6 +450,16 @@
           "default": false,
           "description": "Run 'go test -coverprofile' on save"
         },
+        "go.showCovered": {
```

Comment:
no problem, I am planning to release an update early next week. So Friday sounds good.